modules/Story/Story.tsx

```diff
@@ -40,35 +40,34 @@ function Story({ story }: Props) {
         <Layout>
             <StorySeo story={story} />
             <article className={styles.story}>
-                {headerImage && (
-                    <Image
-                        alt=""
-                        className={classNames({
-                            [styles.mainImage]: !IS_FANCY_IMAGE_ENABLED,
-                            [styles.fullWidthImage]: IS_FANCY_IMAGE_ENABLED,
-                        })}
-                        objectFit="cover"
-                        layout="fill"
-                        imageDetails={headerImage}
-                    />
-                )}
                 <div
                     className={classNames(styles.container, {
                         [styles.withImage]: hasHeaderImage && !IS_FANCY_IMAGE_ENABLED,
                         [styles.withFullWidthImage]: hasHeaderImage && IS_FANCY_IMAGE_ENABLED,
                     })}
                 >
-                    <StoryStickyBar story={story} />
-                    {isEmbargoStory(story) && <Embargo story={story} />}
-                    <div className={styles.meta}>
-                        {showDate && <StoryPublicationDate story={story} />}
-                        {showDate && hasCategories && <> &middot; </>}
-                        {hasCategories && (
-                            <CategoriesList categories={categories} showAllCategories />
-                        )}
-                    </div>
+                    {hasCategories && <CategoriesList categories={categories} showAllCategories />}
                     <h1 className={styles.title}>{title}</h1>
                     <p className={styles.subtitle}>{subtitle}</p>
+                    {showDate && (
+                        <p className={styles.date}>
+                            <StoryPublicationDate story={story} />
+                        </p>
+                    )}
+                    <StoryLinks url={links.short || links.newsroom_view} />
+                    {headerImage && (
+                        <Image
+                            alt=""
+                            className={classNames({
+                                [styles.mainImage]: !IS_FANCY_IMAGE_ENABLED,
+                                [styles.fullWidthImage]: IS_FANCY_IMAGE_ENABLED,
+                            })}
```

Comment:
I guess we're not going with the full width image option after all, so these conditional styles can probably be removed (along with the `fullWidthImage` class in the stylesheet and the constant on top).

@iamhugofelix let us know if you think this might be used later down the road.